src/test/java/com/plaid/client/integration/AccountsBalanceGetTest.java

```diff
@@ -39,7 +39,7 @@ public void testAccountsBalanceGetSuccess() throws Exception {
 
     // sandbox should return expected accounts
     List<Account> accounts = response.body().getAccounts();
-    assertEquals(4, accounts.size());
+    assertEquals(5, accounts.size());
```

Comment:
I believe this change is related to the new account added to the sandbox environment.